doc/main.md

```diff
@@ -1,8 +0,0 @@
-Common Partial-Wave-Analysis Framework (ComPWA) {#mainpage}
-===========================================================
-
-Welcome to the class documentation of ComPWA!
-
-These pages are useful if you are working on the C++ backend of the %ComPWA framework. See [the repository on GitHub](https://github.com/ComPWA/ComPWA) for the source code.
-
-ComPWA is best used through it's python interface (`pycompwa`). You can find all installation instructions and usage examples on our website, [compwa.github.io](https://compwa.github.io/).
```

Comment:
I currently dont understand what the doxygen documentation will show as a mainpage.

Reply:
Ah good point, sorry that was removed in an earlier version

Reply:
https://github.com/ComPWA/ComPWA/pull/312/commits/89d866caf8a517870eda5f2a5150843d863248ad